decidim-assemblies/app/forms/decidim/assemblies/admin/assembly_form.rb

```diff
@@ -50,10 +50,14 @@ def scope
           @scope ||= current_organization.scopes.where(id: scope_id).first
         end
 
+        def to_param
+          id
```

Comment:
This attribute is not defined, isn't it? I think you meant `context[:assembly_id]`, right?

Reply:
This code was being used, actually, so 🔥